docs/source/install/install.rst

```diff
@@ -108,39 +100,20 @@ permissions, you may need to run the last command with :command:`sudo`.
 Windows
 -------
 
-As mentioned above, on Windows, IPython requires :mod:`setuptools`, and it also
-requires the PyReadline library to properly support coloring and keyboard
+As mentioned above, terminal IPython requires the PyReadline library to properly support coloring and keyboard
 management (features that the default windows console doesn't have).  So on
 Windows, the installation procedure is:
 
-1. Install `setuptools <http://pypi.python.org/pypi/setuptools>`_.
-
-2. Install `pyreadline <http://pypi.python.org/pypi/pyreadline>`_.  You can use
-   the command ``pip install pyreadline`` from a terminal, or the binary
-   installer appropriate for your platform from the PyPI page.
-
-3. Install IPython itself, which you can download from `PyPI
-   <http://pypi.python.org/pypi/ipython>`_ or from `our site
-   <http://ipython.org/download.html>`_.  Note that on Windows 7, you *must*
-   right-click and 'Run as administrator' for the Start menu shortcuts to be
-   created.
+1. Install `pip <http://pypi.python.org/pypi/pip>`_, if you don't have it already.
+2. Install IPython: ``pip install ipython``.
```

Comment:
right, this section was supposed to be about installing from source without pip for some reason. I would be inclined to remove the section, then, since Windows is no longer different from other systems in that case. What do you think?